src/domains/shared-app/99_variables.tf

```diff
@@ -132,3 +132,21 @@ variable "tls_cert_check_helm" {
   })
   description = "tls cert helm chart configuration"
 }
+
+variable "influxdb2_helm" {
+  type = object({
+    chart_version = string,
+    image = object({
+      name = string,
+      tag  = string
+    })
+  })
+  default = {
```

Comment:
as suggest in to gilda meeting consider using version 1 to avoid
  https://github.com/grafana/xk6-output-influxdb